bugbug/utils.py

```diff
@@ -513,3 +513,10 @@ def escape_markdown(text: str) -> str:
         .replace(")", "\\)")
         .replace("|", "\\|")
     )
+
+
+def get_business_days_count(start_date, end_date):
+    """Removes weekends from date range."""
+    np_start_date = np.datetime64(start_date[:10], "D")
+    np_end_date = np.datetime64(end_date[:10], "D")
+    return np.busday_count(np_start_date + 1, np_end_date + 1) * 1.0
```

Comment:
I initially had it that way, since the original function, dividing with `.total_seconds()`, resulted in a float, and I wasn't sure if anything depended on it being stored that way. I didn't *think* it was necessary, though, so I'll go ahead and switch it.